### ISSUE TITLE
VAMB issues

### ISSUE THREAD

**Issue by mpdoane2** [linsalrob/atavide_lite#1]
The VAMB version in the atavide_lite_vamb.yaml needs to be updated to 3.0.2 for use GPU computing. This update however might also create some issues with the current minimap/vamb implementation. The error when running vamb is a list of this error for each file:

[E::idx_find_and_load] Could not retrieve index file for 'vamb/mapped_reads/NFI_15tigershark_S7.bam'

When looking into the vamb scripts, two of the same minimap commands occur. Could this be causing the index file to be lost and throw this error?

vamb_minimap.slurm:

minimap2 -t 16 -N 5 -ax sr $BGFS/$OUTDIR/contigs.mmi --split-prefix $BGFS/mmsplit$$ $BGFS/fastq/$R1 $BGFS/fastq/$R2 | samtools view -F 3584 -b --threads 16  | samtools sort -@ 16 -o $BGFS/$OUTDIR/mapped_reads/$BAM -

vamb.slurm
minimap2 -t 16 -N 5 -ax sr $BGFS/$OUTDIR/contigs.mmi --split-prefix $BGFS/mmsplit$$ $BGFS/fastq/$R1 $BGFS/fastq/$R2 | samtools view -F 3584 -b --threads 16  | samtools sort -@ 16 -o $BGFS/$OUTDIR/mapped_reads/$BAM -

**Comment by linsalrob** [comment]
You need to update your `vamb` installation using the [atavide_lite_vamb](https://github.com/linsalrob/atavide_lite/blob/main/atavide_lite_vamb.yaml) description (which just uses the latest version of vamb). 

It is likely that your installation is out of date, and there are a few ways to update it. However, first I recommend cleaning your `conda` package directory (this will also free up some space on your harddrive!)

```
conda clean -af
```

Next, there are plenty of ways to update the code. Choose the arrows to learn more.

<details>
<summary>
1. update the existing installation.
</summary>

Activate your environment, and then update vamb.

```
mamba activate atavide_lite_vamb
mamba update vamb
```
</details>
<details>
<summary>
2. Delete your existing environment and start again
</summary>

```
conda env remove --name atavide_lite_vamb
mamba env create -f atavide_lite_vamb.yaml
```
</details>
<details>
<summary>
3. Create a temporary installation somewhere else and use that.
</summary>

If you are not sure, this is a good way to make a temporary installation of vamb somewhere else and use that in your code.

```
mamba env create -p /tmp/vamb -f atavide_lite_vamb.yaml
mamba activate /tmp/vamb
```

Then you can delete this when you are done!

</details>

Bottom line is that you have an out-of-date vamb installation somewhere, and hopefully you can update it with one of these options.
